### PR TITLE
Using log instead of println

### PR DESCRIPTION
**Pull request by ChengYueJia**
see more on #342 

see more on old pr #921

### CODE REVIEW

File: zkevm-circuits/src/evm_circuit.rs [hunk 1]
```diff
@@ -513,13 +514,13 @@ mod evm_circuit_stats {
             }
         }
 
-        println!(
+        info!(
```
Comment (CPerezz) [inline zkevm-circuits/src/evm_circuit.rs:517]: I think we agreed to remove the logs from the table prints as they were in the tests.

**Review by CPerezz — CHANGES_REQUESTED** [review]
The MD tables should nit be printed with logs. Nor any test-only printed thing.

Aside from that, LGTM.

File: zkevm-circuits/src/evm_circuit.rs (hunk 1 above)
Comment (ChengYueJia) [reply zkevm-circuits/src/evm_circuit.rs]: Year, Sorry for that. I've done it now

**Comment by lispc** [comment]
println should not be replaced by log in build.rs? 

https://doc.rust-lang.org/cargo/reference/build-scripts.html

**Comment by ChengYueJia** [comment]
> println should not be replaced by log in build.rs?

Yeah, make sense.

**Review by CPerezz — APPROVED** [review]
LGTM modulo tests passing.

**Pull request merged**